doc/features/metrics/app-metrics/README.md

```diff
@@ -0,0 +1,140 @@
+# DataStax Metrics Provider based on App.Metrics
+
+This documentation refers to our provider based on App.Metrics. The code examples shown here only apply if this provider is used.
+
+## Installation
+
+This provider is available on the [CassandraCSharpDriver.AppMetrics nuget package].
+
+```
+PM> Install-Package CassandraCSharpDriver.AppMetrics
+```
+
+## Using this provider to enable metrics
+
+This package exposes an extension method of `App.Metrics.IMetricsRoot` that can be used to create the provider:
+
+```csharp
+var metricsRoot = new App.Metrics.MetricsBuilder().Build();
+
+var cluster = Cluster
+    .Builder()
+    .AddContactPoint("127.0.0.1")
+    .WithMetrics(metricsRoot.CreateDriverMetricsProvider())
+    .Build();
+```
+
+## Retrieving metrics
+
+### Retrieving metrics with the provider's interfaces
+
+The driver's interfaces for each metric type are empty so, in order to be able to retrieve the metric values, we have to retrieve the metric objects with the appropriate provider type.
```

Comment:
Same as above, we can recommend again to output using a reporter first, I would order the topic of "Exporting" before "Retrieving" :)

Reply:
done